os/linux_base.h

```diff
@@ -13,6 +13,15 @@
 #ifndef __OS_LINUX_BASE__
 #define __OS_LINUX_BASE__
 
+#if __has_include(<config/config_ac.h>)
+#include <config/config_ac.h>
+#else
+#include <config/config.h>
```

Comment:
FWIW the reason for the name change of the autotools-generate header was to avoid overwriting the manually maintained config/config.h header for Darwin (and getting it to show up as changed in git etc).

it looks like the same thing was replicated with the CMake build system, so only `#include <config/config_ac.h>` should be needed here ?